process-optaplanner-quarkus/pom.xml

```
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-examples</artifactId>
+    <version>8.0.0-SNAPSHOT</version>
+  </parent>
+  <groupId>org.kie.kogito.examples</groupId>
+  <artifactId>process-optaplanner-quarkus</artifactId>
+
+  <name>Kogito Example :: jBPM, OptaPlanner and Quarkus</name>
```

```suggestion
  <name>Kogito Example :: Process, OptaPlanner and Quarkus</name>
```

Please also look up after this word in the README and any docs in this example.